### PR TITLE
Add bias argument to LPLN

### PR DESCRIPTION
**Pull request by mvpatel2000**
# What does this PR do?

Add bias argument to LPLN and forward to LayerNorm parent class

### CODE REVIEW

File: composer/algorithms/low_precision_layernorm/low_precision_layernorm.py [hunk 1]
```diff
@@ -75,11 +75,12 @@ def apply(self, event: Event, state: State, logger: Logger) -> Optional[int]:
 
 class LPLayerNorm(torch.nn.LayerNorm):
 
-    def __init__(self, normalized_shape, eps=1e-05, elementwise_affine=True, device=None, dtype=None):
+    def __init__(self, normalized_shape, eps=1e-05, elementwise_affine=True, bias=True, device=None, dtype=None):
```
Comment (dakinggg) [inline composer/algorithms/low_precision_layernorm/low_precision_layernorm.py:78]: Want to add types while you're here?

**Review by dakinggg — APPROVED** [review]

File: composer/algorithms/low_precision_layernorm/low_precision_layernorm.py (hunk 1 above)
Comment (dakinggg) [inline composer/algorithms/low_precision_layernorm/low_precision_layernorm.py:78]: Also probably need to put bias at the end to keep arg order?

Comment (mvpatel2000) [reply composer/algorithms/low_precision_layernorm/low_precision_layernorm.py]: I'm following torch order

Comment (mvpatel2000) [reply composer/algorithms/low_precision_layernorm/low_precision_layernorm.py]: I think better to be consistent

**Comment by vchiley** [comment]
only relevant in `torch.__version__ >= 2.1`; should we check that?

**Pull request merged**